src/main/webapp/app/lecture/manage/lecture-units/attachment-video-unit-form/attachment-video-unit-form.component.ts

```
@@ -187,17 +208,27 @@ export class AttachmentVideoUnitFormComponent implements OnChanges {
         return this.form.get('urlHelper');
     }
 
+    get videoTranscriptionControl() {
+        return this.form.get('videoTranscription');
+    }
+
     submitForm() {
         const formValue = this.form.value;
         const formProperties: FormProperties = { ...formValue };
+
+        delete formProperties.videoTranscription;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)